apps/arweave/src/ar.erl

```diff
@@ -84,12 +84,39 @@ show_help() ->
 					") where packing is either a mining address or \"unpacked\"."
 					" Example: storage_module 0,En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI. "
 					"To configure a module of a custom size, set "
-					"storage_module [number],[size_in_bytes],[packing]. For instance, "
+					"storage_module {number},{size_in_bytes},{packing}. For instance, "
 					"storage_module "
 					"22,1000000000000,En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI will be "
 					"syncing the weave data between the offsets 22 TB and 23 TB. Make sure "
 					"the corresponding disk contains some extra space for the proofs and "
-					"other metadata, about 10% of the configured size."},
+					"other metadata, about 10% of the configured size."
+
+					"You may repack a storage module in-place. To do that, specify "
+					"storage_module "
+					"{partition_number},{packing},repack_in_place,{target_packing}. "
+					"For example, if you want to repack a storage module "
+					"22,En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI to the new address "
```

Comment:
Is this correct?

`storage_module 22,En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI,repack_in_place,Q5EfKawrRazp11HEDf_NJpxjYMV385j21nlQNjR8_pY`

will load data from 

`data_dir/storage_modules/storage_module_22_En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI`

repack it to `Q5EfKawrRazp11HEDf_NJpxjYMV385j21nlQNjR8_pY` and then write it back to `data_dir/storage_modules/storage_module_22_En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI`

Once once the repacking is complete the miner will need to rename `data_dir/storage_modules/storage_module_22_En2eqsVJARnTVOSh723PBXAKGmKgrGSjQ2YIGwE_ZRI` to `data_dir/storage_modules/storage_module_22_Q5EfKawrRazp11HEDf_NJpxjYMV385j21nlQNjR8_pY`

Reply:
Yes